src/stats/block0/voters.rs

```diff
@@ -0,0 +1,49 @@
+use chain_crypto::bech32::Bech32;
+use jormungandr_automation::{jormungandr::JormungandrRest, testing::block0::get_block};
+use jormungandr_lib::interfaces::Initial;
+
+pub fn count_active_voters<S: Into<String>>(endpoint: S) -> Result<(), crate::stats::Error> {
+    let endpoint = endpoint.into();
+    let block0_path = format!("{}/v0/block0", &endpoint);
+    println!("Reading block0 from location {:?}...", block0_path);
+    let genesis = get_block(block0_path)?;
+
+    let rest_client = JormungandrRest::new(endpoint);
+    let mut stats: Stats = Default::default();
+
+    let mut total = 0;
+    for initial in genesis.initial.iter() {
+        if let Initial::Fund(initial_utxos) = initial {
+            total += initial_utxos.len();
+        }
+    }
+
+    println!("total: {}", total);
```

Comment:
I don't like important information from a library function to be conveyed by printing to stdout. This should be in the return value instead

Reply:
Command removed, since there is no big business case.